spec/models/task_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Task, type: :model do
```

Comment:
Just get rid of this file - if it's not doing anything we don't need it. Integration tests are more useful at this stage.